README.md

```
@@ -153,6 +153,20 @@ dot -Tpdf cfg.dot > cfg.pdf
 
+## Running unit tests
+
+Build with `VERIFIER_ENABLE_TESTS=ON` to produce the `tests` binary. Run the
+suite sequentially using:
+```bash
+./tests -d yes
+```
+
+For faster execution, `scripts/run-tests-parallel.sh` splits the Catch2 suite
+across available CPUs and runs the shards concurrently:
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)